pkg/config.go

```diff
@@ -0,0 +1,36 @@
+package pkg
+
+
+// this object should be responsible for
+// loading file from external resources /xml/json/yaml
+type RulesContainer struct {
+	ruleMap map[string]map[string]int
```

Comment:
`map[string]map[string]int` - not so understable what is used for. Map of what of what?
Either add a comment, either use custom type (where e.g. map[string]int will be `QuotaRule`, or something like that)

Reply:
you are right, here should be comment, if it will be bigger than would be case to move into another class